src/modules/navigator/rtl_params.c

```diff
@@ -173,3 +173,15 @@ PARAM_DEFINE_INT32(RTL_PLD_MD, 0);
  * @group Return Mode
  */
 PARAM_DEFINE_FLOAT(RTL_LOITER_RAD, 50.0f);
+
+/**
+ * RTL heading mode
+ *
+ * Allow heading in direction of navigation, use current, or use destination heading during transition to destination position.
```

Comment:
Defines the behavior 
```suggestion
 * Defines the heading behavior during RTL
```